### PR TITLE
Make a chart of REX events by dorm

### PR DESCRIPTION
**Pull request by camtheman256**
I'm not saying this is a good idea, but it sure is funny to have on the website.

Also open to feedback for what should go on the chart.

image

### CODE REVIEW

**Review by dtemkin1 — APPROVED** [review]
LMAO I love it, looks good to me >:)

I wonder if there's any way to combine the cultural houses and the New House entry? If not its fine, but it might be good

**Comment by camtheman256** [comment]
> LMAO I love it, looks good to me >:)
> 
> I wonder if there's any way to combine the cultural houses and the New House entry? If not its fine, but it might be good

screw it, lemme try actually

**Comment by camtheman256** [comment]
done lmfao

Screenshot 2023-08-14 at 20 22 08

**Comment by dtemkin1** [comment]
slayyyy 🫡🫡🫡

**Comment by camtheman256** [comment]
i am worried we are kinda j shaming baker but i feel like that's fine actually

File: components/t-rex/RexEventChart.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    const [api, setApi] = useState<TRexAPIResponse>();
+
+    useEffect(() => {
+        fetchEvents().then((d) => {
+            setApi(d);
+            const byDorm = new Map<string, number>();
+            for (const event of d.events) {
+                let dorm = event.dorm;
+                if (
+                    [
```
Comment (dtemkin1) [inline components/t-rex/RexEventChart.tsx:39]: dont forget chocolate city (they didnt submit any events, but just in case for future use ig)

**Comment by dtemkin1** [comment]
> i am worried we are kinda j shaming baker but i feel like that's fine actually

if they dont want to be shamed they should hold more events <3

**Pull request merged**